### ISSUE TITLE
CNTextInput additional props can not be added 

### ISSUE THREAD

**Issue by pacozaa** [imnapo/react-native-cn-richtext-editor#97]
CNTextInput additional props can not be added. There should be something like `{...props}` to pass additional props.

**Comment by KristineTrona** [comment]
Indeed, in my case I find it very annoying that I cannot pass a prop to disable the autoCorrect functionality on the TextInput

**Comment by pacozaa** [comment]
@KristineTrona I have worked out with the contributors in this repo about the issue. Please check out his new version and document. Hope you find something useful.

**Comment by KristineTrona** [comment]
@pacozaa Thanks for the reply, but I am already on the latest release 1.0.42 and I also checked out the pre-release code/master branch and I do not see the possibility to pass any additional props to the TextInput component. I hope you are correct and they will add it soon.

**Comment by pacozaa** [comment]
@KristineTrona Is the problem at `CNTextInput.js`? Do you have example code or snippet that you wanna use through the library?

**Comment by KristineTrona** [comment]
@pacozaa I've just created a PR #101 for passing autoCorrect prop to the textInput, but I do feel like a more general option of prop spreading like what you suggested with ``{...props}`` would be better
